docs/user/gcp/install_upi.md

```diff
@@ -0,0 +1,507 @@
+example.
+
+## Create Ignition Configs
+
+The machines will be started manually. Therefore, it is required to generate
+the bootstrap and machine Ignition configs and store them for later steps.
+Use [a staged install](../overview.md#multiple-invocations) to remove the
+control-plane Machines and compute MachineSets, because we'll be providing
+those ourselves and don't want to involve
+[the machine-API operator][machine-api-operator].
+
+```console
+$ openshift-install create install-config
+? SSH Public Key /home/user_id/.ssh/id_rsa.pub
+? Platform gcp
+? Project ID example-project
+? Region us-east1
+? Base Domain example.com
+? Cluster Name openshift
+? Pull Secret [? for help]
+```
+
+Edit the resulting `openshift-install.yaml` to set `replicas` to 0 for the `compute` pool:
+
+```console
+$ sed -i '1,/replicas: / s/replicas: .*/replicas: 0/' install-config.yaml
+```
+
+Create manifests to get access to the control-plane Machines and compute MachineSets:
+
+```console
+$ openshift-install create manifests
+INFO Consuming "Install Config" from target directory
+```
+
+From the manifest assets, remove the control-plane Machines and the compute MachineSets:
+
+```console
+$ rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml openshift/99_openshift-cluster-api_worker-machinesets-*.yaml
+```
+
+You are free to leave the compute MachineSets in if you want to create compute
+machines via the machine API, but if you do you may need to update the various
+references (`subnet`, etc.) to match your environment.
+
+Now we can create the bootstrap Ignition configs.
+
+```console
+$ openshift-install create ignition-configs
+```
+
+After running the command, several files will be available in the directory.
+
+```console
+$ tree
+.
+├── auth
+│   └── kubeconfig
+├── bootstrap.ign
+├── master.ign
+├── metadata.json
+└── worker.ign
+```
+
+### Extract Infrastructure Name from Ignition Metadata
+
+By default, Ignition generates a unique cluster identifier comprised of the
+cluster name specified during the invocation of the installer and a short
+string known internally as the infrastructure name (infra_id). These values
+are seeded in the initial manifests within the Ignition configuration. To use
+the output of the default, generated `ignition-configs` extracting the internal
+infrastructure name is necessary.
+
+An example of a way to get this is below:
+
+```
+$ jq -r .infraID metadata.json
+openshift-vw9j6
+```
+
+Export infra_id and several other variables to be used in examples below.
+
+```console
+export BASE_DOMAIN='example.com'
+export BASE_DOMAIN_ZONE_NAME='example'
+export CLUSTER_NAME=`jq -r .clusterName metadata.json`
+export INFRA_ID=`jq -r .infraID metadata.json`
+export PROJECT_NAME=`jq -r .gcp.projectID metadata.json`
+export REGION=`jq -r .gcp.region metadata.json`
+```
+
+## Create the VPC to be Used
+
+Copy the [template](../../../upi/gcp/01_vpc.py) locally
+and create a resource definition file: 01_vpc.yaml
```

Comment:
nit: backticks around the path literal: `01_vpc.yaml`

Reply:
or you could just use:

```markdown
Save the [`01_vpc.py`](../../../upi/gcp/01_vpc.py) locally.
```

Reply:
Good ideas. I added the backticks to the yaml file names, and used the python file names in the links.